app/views/transactions/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<p style="color: green"><%= notice %></p>
+
+<div id="transations" class=" ">
+  <% @transactions.each do |transaction| %>
+    <%= render transaction %>
+  <% end %>
+</div>
```

Comment:
- [ ] kindly make sure that on your transaction page for a given category, the list of transactions is presented, ordered by the most recent, at the top of the page, the user can see the total amount for the category (sum of all of the amounts of the transactions in that category) and there should be a button "add a new transaction" at the bottom that brings the user to the page to create a new transaction :blush: 
  
  **Currently, you only have a navigation bar on your transaction page when I click on a category**

  ![image](https://github.com/emarkees/New_Budget_App/assets/92303891/8824637f-58ac-4ac6-8784-552ba43bd2f6)

Reply:
done